docusaurus.config.js

```diff
@@ -0,0 +1,97 @@
+// @ts-check
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Stencil',
+  tagline: 'A powerful toolchain for building Progressive Web Apps and Design Systems',
+  url: 'https://stenciljs.com',
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'icon/favicon.ico',
+
+  organizationName: 'ionic-team',
+  projectName: 'stencil-site',
+
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
+  },
+
+  scripts: [
+    {
+      src: "https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js",
+      async: true,
+    }
+  ],
```

Comment:
scripts set here are added to the `<head>` element for pages rendered by Docusaurus, see https://docusaurus.io/docs/api/docusaurus-config#scripts